src/lib/stores/modals.svelte.ts

```diff
@@ -1,10 +1,10 @@
-import { Map } from "svelte/reactivity";
+import {  SvelteMap } from "svelte/reactivity";
```

Comment:
I think you forgot to remove this.
just build-in `Map` is good :)

Reply:
Should I replace `SvelteMap` with deprecated `Map`, Some how it is not even loading the mock data for me

Reply:
Should I replace SvelteMap with deprecated Map, Some how it is not even loading the mock data for me

Reply:
No? we can use javascript build-in Map function

Reply:
In this case @meowsocute  is correct. This should be `SvelteMap`